src/ui/components/shared/Forms/Checkbox.tsx

```diff
@@ -4,12 +4,14 @@ import React from "react";
 export default function Checkbox({
   id,
   checked,
+  disabled,
   onChange,
   className,
 }: React.HTMLProps<HTMLInputElement>) {
   return (
     <input
       type="checkbox"
+      disabled={disabled}
```

Comment:
Btw I don't think this component is used in the new code at all. The change seems fine though, so long as it was intentional.

Reply:
Yeah, this was the old `Checkbox` I was trying to import. I moved things to a new one, but will leave this since it doesn't harm anything.